liveCricketScore.py

```diff
@@ -0,0 +1,24 @@
+import requests
+from bs4 import BeautifulSoup
+from time import sleep
+
```

Comment:
Import statements for BeauitfulSoup and requests are missing.

Reply:
I am not getting what you are trying to say. I did import the BeautifulSoup module from bs4 and also the requests too.